core/math/color.cpp

```diff
@@ -432,6 +432,26 @@ Color Color::get_named_color(int p_idx) {
 	return named_colors[p_idx].color;
 }
 
+String Color::to_named() const {
+	NamedColor closest_named_color = {};
+	float closest_distance_squared = std::numeric_limits<float>::infinity();
+
+	for (NamedColor named_color : named_colors) {
```

Comment:
Use a const reference to avoid unnecessary copies:

```suggestion
	for (const NamedColor &named_color : named_colors) {
```